openhtf/core/test_executor.py

```diff
@@ -51,7 +51,7 @@ class TestExecutor(threads.KillableThread):
   """Encompasses the execution of a single test."""
   daemon = True
 
-  def __init__(self, test_descriptor, execution_uid, test_start,
+  def __init__(self, test_descriptor, execution_uid, test_start, default_dut_id,
```

Comment:
Is there a reason we aren't just passing in a TestOptions here instead of unpacking all the values to pass them in one at a time?

Reply:
That seems like a good change we should make.